.github/workflows/test-integrations-web-frameworks-1.yml

```diff
@@ -64,6 +64,7 @@ jobs:
       - name: Test django latest
         run: |
           set -x # print commands that are executed
+          export PIP_CONSTRAINT=constraints.txt
```

Comment:
These CI files are auto generated by [this script](https://github.com/getsentry/sentry-python/blob/master/scripts/split-tox-gh-actions/split-tox-gh-actions.py) from [this template](https://github.com/getsentry/sentry-python/blob/master/scripts/split-tox-gh-actions/templates/test_group.jinja), so you will need to update the template to include this change and then run `split-tox-gh-actions.py` to generate the files.